ex1/README.md

```diff
@@ -0,0 +1,17 @@
+# Welcome to monday-u!
```

Comment:
ok seems like all of the other exercises are committed here as well, next time separate them to another PR